### ISSUE TITLE
Batch Normalization - Jingyu Xiong's Blog

### ISSUE THREAD

**Issue by jyxiong** [jyxiong/jyxiong.github.io#19]
https://jyxiong.github.io/2018/03/05/Batch-Normalization.html 

 ## BGDSGDMBGD
